lib/usb-device.js

```diff
@@ -0,0 +1,237 @@
+// import constants
+import { USB_VENDOR_IDS, ADB_VALUES, CONNECT_VALUES,
+          ADB_COMMANDS, ADB_KEY } from './constants';
+let MAXDATA = CONNECT_VALUES.CONNECT_MAXDATA;
+// import helpers
+import { generateMessage, packetFromBuffer } from './helpers';
+
+// required libraries
+import path from 'path';
+import fs from 'fs';
+import Promise from 'bluebird';
+import * as usb from 'usb';
+Promise.promisifyAll(usb);
```

Comment:
it's unclear why you're promisifying `usb`. AFAICT you don't `await usb.anything`. I took a brief brief look at the usb docs and they seemed to be sync functions (i.e., no need to await or promisify) but I could be wrong.